### PR TITLE
Clarifying that you need to take a manual action to trigger hot reload

### PR DESCRIPTION
**Pull request by sethladd**
Addressing https://github.com/flutter/flutter/issues/8717 and https://github.com/flutter/flutter/issues/8717#issuecomment-289892559

### CODE REVIEW

**Comment by sethladd** [comment]
cc @InMatrix

File: getting-started.md [hunk 1]
```diff
@@ -78,19 +78,22 @@ If everything works, you should see your starter app on your device or simulator
...
+in your simulator, emulator, or device.
 
 ![Hot reload in IntelliJ](/images/intellij/hot-reload.gif)
 
 To edit your code and hot reload changes:
 
 1. Run your app with 'Run' or 'Debug'.
 1. Make desired changes in your source code (without stopping the app).
-1. Click the Hot Reload button.
+1. Click the Hot Reload button (the button with the lightning bolt and the green run triangle).
```
Comment (InMatrix) [inline getting-started.md]: The button is hard to describe is part of the problem. :-) Hopefully, we'll have a simpler icon for hot reload.

**Comment by InMatrix** [comment]
LGTM. Thanks for making this quick change.

File: getting-started.md (hunk 1 above)
Comment (sethladd) [inline getting-started.md]: happy to update this later, when the UI changes

Comment (Sfshaza) [inline getting-started.md]: +1 to a new icon!  I just tried to launch IntelliJ CE to copy the icon, but it's the 2017.1 version doesn't yet support Flutter.

Comment (mit-mit) [inline getting-started.md]: We will likely change the icon fairly soon, but for now this description looks good!

**Comment by mit-mit** [comment]
LGTM

**Pull request merged**